tests/unit/test_dojson.py

```diff
@@ -0,0 +1,379 @@
+# -*- coding: utf-8 -*-
```

Comment:
Better to use `conftest.py`
Where are the `pjson` dump and `to_marc21`transformation for better testing ?

Reply:
it is back again